app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -786,6 +789,62 @@ class Settings(private val appContext: Context) : PreferencesHolder {
             0
         )
 
+    private val numTimesFeedbackPromptShown: Int
```

Comment:
a-c has preference helpers to help with these!

`private val numTimesFeedbackPromptShown by intPreference(key, defaultValue)`